README.md

```diff
@@ -73,5 +73,14 @@ $ docker-compose -f docker-compose.prod.yml up
 ```
 It will serve backend at port 8080 and frontend at port 80.
 
+
+## Mobile
+
+For development:
+```yarn start```
```

Comment:
In my case, I had to run 'sudo yarn start' instead. Running it without root privileges raised an error:
```
Error: EACCES: permission denied, open '/home/rafal/testinno/mobile/node_modules/.cache/@babel/register/.babel.7.3.4.development.json'
```